.github/workflows/ci.yml

```
@@ -12,6 +12,10 @@ jobs:
           node-version-file: .nvmrc
           cache: 'npm'
       - name: Install dependencies
-        run: npm ci
+        run: |
+          npm ci
+          npx playwright install-deps
       - name: Lint (JavaScript)
         run: npm run lint
+      - name: Unit Tests
+        run: npm run test
```

```suggestion
      - name: Lint (Javascript) and Unit Tests
        run: npm run test
```

```suggestion
      - name: Lint (JavaScript)
        run: npm run lint
      - name: Unit Tests (Server)
        run: npm run test:server
```
 Since `npm run test` includes linting, maybe we just remove that step?  Or if we want to keep them separate, we can add separate steps for the server tests (and then eventually the browser tests)?